girder/utility/assetstore_utilities.py

```diff
@@ -60,6 +59,15 @@ def getAssetstoreAdapter(assetstore, instance=True):
         return cls
 
 
+def setAssetstoreAdapter(storeType, cls):
+    global assetstoreTable
+
+    if storeType not in assetstoreTable:
+        raise Exception('Illegal assetstore type code: "%s"' % (storeType))
```

Comment:
I think this is actually fine, I'd like for plugins to be able to call this as a way to register new assetstore adapters as opposed to using the old `assetstore.adapter.get` hook.